__tests__/app.test.js

```diff
@@ -425,5 +426,33 @@ describe('/api/articles/?=:topic)', () => {
       })
   })
 
+  })
+})
+
+describe('/api/articles/:article_id(comment_count)', () => {
+  test('GET 200: response object now includes total number of comments for article_id', () => {
+    return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article.article_id).toBe(3)
+            expect(response.body.article.author).toBe("icellusedkars")
+            expect(response.body.article.title).toBe("Eight pug gifs that remind me of mitch")
+            expect(response.body.article.body).toBe("some gifs")
+            expect(response.body.article.topic).toBe("mitch")
+            expect(response.body.article.votes).toBe(0)
+            expect(response.body.article.created_at).toBe("2020-11-03T09:12:00.000Z")
+            expect(response.body.article.article_img_url).toBe( "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
```

Comment:
I mean this does work but not very nice to look at! `toMatchObject` will mae this so much nicer! 😁